packages/mcp-gmail/docker-compose.yml

```
@@ -0,0 +1,41 @@
+      context: ../..  # Build from workspace root
+      dockerfile: packages/mcp-gmail/Dockerfile
+    container_name: gmail-mcp-server
+    restart: unless-stopped
+    ports:
+      - "3001:3000"  # Map to different port to avoid conflicts with RAG
+    environment:
+      # Required for cloud mode
+      - USE_LOCAL_GMAIL_SERVER=false
+      - NODE_ENV=production
+      - PORT=3000
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)